### DISCUSSION TITLE
Social Procurement Definition

### DISCUSSION THREAD

**Discussion by jseguraf** [eprocurementontology/eprocurementontology#514] [Ideas]
Proposed definition: 

Socially responsible public procurement (SRPP) means procurement operations that take into account one or more of the following social considerations: employment opportunities, decent work, compliance with social and labour rights, social inclusion (including persons with disabilities), equal opportunities, accessibility design for all, taking account of sustainability criteria, including ethical trade issues and wider voluntary compliance with corporate social responsibility (CSR), while observing the principles enshrined in the Treaty for the European Union (TFEU) and the Procurement Directives. 
Source: Buying Social - A Guide to Taking Account of Social Considerations in Public Procurement (Publications Office of the European Union, 2011) https://op.europa.eu/en/publication-detail/-/publication/cb70c481-0e29-4040-9be2-c408cddf081f/language-en
See Article 18 and Annex X Directive 2014/24.

**Comment by VibekeE** [comment]
And from the EC web site https://ec.europa.eu/info/policies/public-procurement/support-tools-public-buyers/social-procurement_en  "“Public authorities can engage in socially-responsible public procurement by buying ethical products and services, and by using public tenders to create job opportunities, decent work, social and professional inclusion and better conditions for disabled and disadvantaged people”.

**Comment by eprocurementontology** [comment]
Socially responsible public procurement (SRPP) as defined by Buying Social:

‘SRPP’ means procurement operations that take into account one or more of the following social considerations: employment opportunities, decent work, compliance with social and labour rights, social inclusion (including persons with disabilities), equal opportunities, accessibility design for all, taking account of sustainability criteria, including ethical trade issues and wider voluntary compliance with corporate social responsibility (CSR), while observing the principles enshrined in the Treaty for the European Union (TFEU) and the Procurement Directives. 
Source: Buying Social - A Guide to Taking Account of Social Considerations in Public Procurement (Publications Office of the European Union, 2011) https://op.europa.eu/en/publication-detail/-/publication/cb70c481-0e29-4040-9be2-c408cddf081f/language-en
See Article 18 and Annex X Directive 2014/24.

Additional information:
An instance of the class SocialProcurement  is represented in eForms with the code "soc-obj" defined in the codelist strategic-procurement.
 https://ec.europa.eu/info/policies/public-procurement/support-tools-public-buyers/social-procurement_en "“Public authorities can engage in socially-responsible public procurement by buying ethical products and services, and by using public tenders to create job opportunities, decent work, social and professional inclusion and better conditions for disabled and disadvantaged people”.
